### PR TITLE
Add Air Europa Conditions of Carriage

### PR DESCRIPTION
**Pull request by clementbiron**

### CODE REVIEW

File: declarations/Air Europa.json [hunk 1]
```diff
@@ -10,6 +10,11 @@
       "fetch": " https://www.aireuropa.com/fr/fr/aea/informations-pour-voler/conditions/conditions-de-reservation.html",
       "select": [".cmp-Jumbo-banner-small", ".cmp-container"],
       "remove": ["#interestBlock"]
+    },
+    "Conditions of Carriage": {
+      "fetch": " https://www.aireuropa.com/fr/fr/aea/informations-pour-voler/conditions/conditions-de-transport.html",
```
Comment (Ndpnt) [inline declarations/Air Europa.json:15]: Extra space to remove

Comment (clementbiron) [reply declarations/Air Europa.json:15]: Thanks !

**Comment by Ndpnt** [comment]
@clementbiron  Can you resolve the conflict?

**Comment by clementbiron** [comment]
@Ndpnt Which one?  👀

**Review by MattiSG — COMMENTED** [review]
There seems to be a problem with the history 🙂
Do you need help @clementbiron?

**Comment by MattiSG** [comment]
@Ndpnt do you have to handle this?

**Comment by MattiSG** [comment]
Same with #316

**Review by martinratinaud — APPROVED** [review]

**Comment by clementbiron** [comment]
> @Ndpnt do you have to handle this?

I handled it.

**Pull request merged**